frontend/src/scenes/annotations/index.tsx

```diff
@@ -208,76 +227,85 @@ function CreateAnnotationModal(props: CreateAnnotationModalProps): JSX.Element {
-                            </Menu>
-                        }
-                    >
-                        <Button style={{ marginLeft: 8, marginRight: 8 }}>
-                            {annotationScopeToName.get(scope)} <DownOutlined />
-                        </Button>
-                    </Dropdown>{' '}
+                    This annotation will appear on all{' '}
+                    <div className="inline-flex">
+                        <LemonSelect
+                            size="small"
+                            dropdownMaxContentWidth={true}
+                            options={{
+                                [AnnotationScope.Project]: { label: 'project', icon: <IconProject /> },
+                                [AnnotationScope.Organization]: {
+                                    label: 'organization',
+                                    icon: <IconOrganization />,
+                                },
+                            }}
+                            onChange={(scope) => {
+                                if (scope) {
+                                    setScope(scope)
+                                }
+                            }}
+                            value={scope}
+                        />
+                    </div>{' '}
                     charts
                 </span>
             ) : (
-                <Row justify="space-between">
-                    <span>Change existing annotation text</span>
+                <div className="flex justify-end">
                     {!props.annotation?.deleted ? (
-                        <DeleteOutlined
-                            className="text-danger"
-                            onClick={(): void => {
-                                props.onDelete()
-                            }}
-                        />
+                        <LemonButton
+                            type="secondary"
+                            status="danger"
+                            onClick={props.onDelete}
+                            icon={<IconDelete />}
+                            data-attr="delete-annotation"
+                        >
+                            Delete annotation
+                        </LemonButton>
                     ) : (
-                        <RedoOutlined
-                            className="button-border clickable"
-                            onClick={(): void => {
-                                props.onRestore()
-                            }}
-                        />
+                        <LemonButton
+                            type="secondary"
+                            status="primary-alt"
+                            onClick={props.onRestore}
+                            icon={<IconUndo />}
+                            data-attr="restore-annotation"
+                        >
+                            Restore annotation
+                        </LemonButton>
```

Comment:
Personally I think a small and simple fix would be to move these to the "tertiary" footer position (on the left).